compiler/src/dotty/tools/dotc/typer/TyperPhase.scala

```
@@ -128,6 +107,12 @@ class FrontEnd extends Phase {
   def run(using Context): Unit = unsupported("run")
 }
 
-object FrontEnd {
+object TyperPhase {
   val name: String = "typer"
 }
+
+@deprecated(message = "FrontEnd has been split into TyperPhase and Parser. Refer to one or the other.")
+object FrontEnd {
```

This is a little sketchy, but should silence the error in the community build.